pkg/apis/resolution/v1beta1/resolution_request_types.go

```diff
@@ -64,6 +64,7 @@ type ResolutionRequestSpec struct {
 	// +optional
 	// +listType=atomic
 	Params []pipelinev1.Param `json:"params,omitempty"`
+	Url    string             `json:"url,omitempty"`
```

Comment:
```suggestion
	URL    string             `json:"url,omitempty"`
```

https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
fixed!